README.md

```diff
@@ -1,71 +1,78 @@
 <div align="center">
-<img src="https://raw.githubusercontent.com/spotifyts/assets/main/svg/full-nobg.svg">
-
-# spotify.ts
+  <img src="https://raw.githubusercontent.com/spotifyts/assets/main/svg/full-nobg.svg" alt="spotify.ts logo">
 
+  # spotify.ts
 </div>
 
 ## About
 
-spotify.ts is an wrapper built around Spotify's Web API.
+`spotify.ts` is a TypeScript wrapper for Spotify's Web API, designed to make interacting with Spotify's services easy and efficient.
 
 ## Features
 
--   Fast
--   Object Oriented
--   Typescript, ESM, CJS support
--   Easy to Use
+- **Fast Performance**
+- **Object-Oriented Design**
+- **Support for TypeScript, ESM, and CJS**
+- **User-Friendly Interface**
+
+## Installation
+
+You can install `spotify.ts` using one of the following package managers:
 
-## Installing
+```bash
+npm install spotify.ts
+```
 
 ```bash
-npm i spotify.ts
 yarn add spotify.ts
+```
+
+```bash
 pnpm add spotify.ts
 ```
 
 ## Usage
 
-First, import `Client` from the library, then instantiate a a new client with the `clientId` and `clientSecret` credentials. Then you can use the managers to make API calls to the Spotify API. For an in-built guide, visit the documentation site (see below).
+To get started, import the `Client` class from the library and create a new instance with your `clientId` and `clientSecret`. You can then use the various managers to interact with the Spotify API. For more detailed instructions, visit the documentation site (link coming soon).
 
-### Examples:
+### Examples
 
-Fetch an album:
+**Fetch an Album:**
 
 ```typescript
 import { Client } from 'spotify.ts';
 
 const client = new Client({
-	clientId: 'some-id',
-	clientSecret: 'some-secret'
+  clientId: 'your-client-id',
+  clientSecret: 'your-client-secret'
 });
 
 client.start().then(() => {
-	client.albums.fetch('0NLkVxf0PyxsXBG3EuZcJf').then(console.log);
+  client.albums.fetch('0NLkVxf0PyxsXBG3EuZcJf').then(console.log);
 });
 ```
 
-Fetch an artist:
+**Fetch an Artist:**
 
 ```typescript
 import { Client } from 'spotify.ts';
 
 const client = new Client({
-	clientId: 'some-id',
-	clientSecret: 'some-secret'
+  clientId: 'your-client-id',
+  clientSecret: 'your-client-secret'
 });
 
 client.start().then(() => {
-	client.artists.fetch('4gdMJYnopf2nEUcanAwstx').then(console.log);
+  client.artists.fetch('4gdMJYnopf2nEUcanAwstx').then(console.log);
 });
 ```
 
 ## Links
 
--   Documentation (Coming Soon)
--   [Discord Server](https://discord.gg/qchtmGDdFr)
--   [Github Organization](https://github.com/spotifyts)
--   [NPM Page](https://npmjs.com/spotify.ts)
+- [Documentation (Coming Soon)](https://github.com/spotifyts/spotify.ts)
```

Comment:
This is meant for the docusaurus documentation site. User's can find their way through the github org link.

Reply:
so shall I remove this it then?

Reply:
yes

Reply:
to this right?
![image](https://github.com/user-attachments/assets/880c757f-fbcc-4043-9a72-75ecc59d37fe)